source/includes/footer.inc.rst

```diff
@@ -2,15 +2,17 @@
 
 .. seealso::
 
-   If you would like to contribute to these docs, but are unsure where to
-   start, please see the |GitHubDocProjectREADME|_ for an overview of the
-   process. If you would like to contribute to the main source project,
-   please review the contribution guidelines listed in the
-   |GitHubSourceProjectREADME|_.
+   Help with configuring/using |PRODUCT|:
 
-   If you have a question about these docs or |PRODUCT| in general, please
-   see the following resources:
+   - |RsyslogOfficialMailingList|_ - best route for general questions
+   - GitHub: |GitHubSourceProject|_ - best route for detailed questions and
+     for reporting issues that are believed to be bugs with |PRODUCT|
+   - |RsyslogStackOverflow|_ - least activity by dev team, but generally
```

Comment:
I would tell that this is *experimental* -- it involves a kind of "use at your own risk, you have been told" kind of attitude IMO

Reply:
I can add further emphasis if you prefer/think it's necessary. I was thinking that "least activity by dev team" would cover that and would still hold true even if we end up providing more support for it over time.

Reply:
I would go for "experimental" as it currently looks pretty seldom that a dev will check, but it's not something I insist on...